tasks-by-users/Kiran-Yadav/01-project-hovercard/effective.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Effective Intelligence</title>
+    <style>
+        *{
+            margin: 0;padding: 0;
+            box-sizing: border-box;
+        }
+        header {
+            width: 100vw;
+            box-sizing: border-box;
+            text-align: center;
+            background-image: url(images/header.jpg);
+            color: wheat;
+            border-radius: 10px;
+            height: 10vh;
+            line-height: 10vh;
+        }
+        main{
+            
+            background: olive;
+            color: #fff;
+            height: 90vh;
+        }
+        main p{
+            margin: 0 10px;
+
+        }
+        footer{
+
+height: 10vh;
+width: 100%;
+background-color: #000;
+text-align: center;
+line-height: 10vh;
+}
+    </style>
```

Comment:
check whether your prettier is working or not? and also remove embedded styling @Kiran-512